### DISCUSSION TITLE
Solved :)

### DISCUSSION THREAD

**Discussion by DeltaFoxtMike** [community/community#17810] [GitHub Education]
.

**Comment by euaaron** [comment]
Hello @AronkyDev, I'm Aaron. I had a similar issue while applying for the first time. I got really frustated because I did delivered all documentations but still got refused again and again. It took near a month until I finally got approved. 

I know how it is to not be approved after some attempts but I would like to say that all your attempts are registed on the system, when someone from GitHub Education team see that you've sent the requested docs and is reeeeally wanting to be part of the program you may be approved.

Even it is a frustrating experience, keep calm, respectful and keep trying. Also, keep being an active user.

Good luck! ✨

> I'm not part of GitHub Education team so everything I've said do not came from any consistent base of truth, they came just from my thoughts, experience and imagination.
